cpp/ql/test/query-tests/Critical/MemoryFreed/UseAfterFree.expected

```diff
@@ -1,100 +1,51 @@
-| test_free.cpp:245:10:245:11 | * ... | test_free.cpp:246:9:246:10 | * ... |
 nodes
 | test_free.cpp:11:10:11:10 | a | semmle.label | a |
-| test_free.cpp:11:10:11:10 | a | semmle.label | a |
 | test_free.cpp:12:5:12:5 | a | semmle.label | a |
-| test_free.cpp:13:6:13:6 | a | semmle.label | a |
 | test_free.cpp:42:27:42:27 | a | semmle.label | a |
-| test_free.cpp:42:27:42:27 | a | semmle.label | a |
-| test_free.cpp:44:27:44:27 | a | semmle.label | a |
 | test_free.cpp:44:27:44:27 | a | semmle.label | a |
 | test_free.cpp:45:5:45:5 | a | semmle.label | a |
 | test_free.cpp:45:5:45:5 | a | semmle.label | a |
 | test_free.cpp:69:10:69:10 | a | semmle.label | a |
-| test_free.cpp:69:10:69:10 | a | semmle.label | a |
 | test_free.cpp:71:9:71:9 | a | semmle.label | a |
 | test_free.cpp:90:10:90:10 | a | semmle.label | a |
-| test_free.cpp:90:10:90:10 | a | semmle.label | a |
 | test_free.cpp:91:5:91:5 | a | semmle.label | a |
 | test_free.cpp:95:10:95:10 | a | semmle.label | a |
 | test_free.cpp:96:9:96:9 | a | semmle.label | a |
 | test_free.cpp:101:10:101:10 | a | semmle.label | a |
 | test_free.cpp:102:23:102:23 | a | semmle.label | a |
 | test_free.cpp:152:27:152:27 | a | semmle.label | a |
-| test_free.cpp:152:27:152:27 | a | semmle.label | a |
 | test_free.cpp:153:5:153:5 | a | semmle.label | a |
 | test_free.cpp:233:14:233:15 | * ... | semmle.label | * ... |
-| test_free.cpp:233:14:233:15 | * ... | semmle.label | * ... |
-| test_free.cpp:236:9:236:10 | * ... | semmle.label | * ... |
 | test_free.cpp:236:9:236:10 | * ... | semmle.label | * ... |
 | test_free.cpp:239:14:239:15 | * ... | semmle.label | * ... |
-| test_free.cpp:239:14:239:15 | * ... | semmle.label | * ... |
-| test_free.cpp:241:9:241:10 | * ... | semmle.label | * ... |
 | test_free.cpp:241:9:241:10 | * ... | semmle.label | * ... |
 | test_free.cpp:241:10:241:10 | b | semmle.label | b |
 | test_free.cpp:245:10:245:11 | * ... | semmle.label | * ... |
-| test_free.cpp:245:10:245:11 | * ... | semmle.label | * ... |
-| test_free.cpp:246:9:246:10 | * ... | semmle.label | * ... |
 | test_free.cpp:246:9:246:10 | * ... | semmle.label | * ... |
 subpaths
 #select
 | test_free.cpp:12:5:12:5 | a | test_free.cpp:11:10:11:10 | a | test_free.cpp:12:5:12:5 | a | Memory may have been previously freed by $@. | test_free.cpp:11:5:11:8 | call to free | call to free |
-| test_free.cpp:12:5:12:5 | a | test_free.cpp:11:10:11:10 | a | test_free.cpp:12:5:12:5 | a | Memory may have been previously freed by $@. | test_free.cpp:11:5:11:8 | call to free | call to free |
-| test_free.cpp:13:6:13:6 | a | test_free.cpp:11:10:11:10 | a | test_free.cpp:13:6:13:6 | a | Memory may have been previously freed by $@. | test_free.cpp:11:5:11:8 | call to free | call to free |
-| test_free.cpp:13:6:13:6 | a | test_free.cpp:11:10:11:10 | a | test_free.cpp:13:6:13:6 | a | Memory may have been previously freed by $@. | test_free.cpp:11:5:11:8 | call to free | call to free |
```

Comment:
We've lost the (correct) result on line 13 here.

Reply:
Uh, that was well spotted. I've fixed this in 1ad3d3a4a61239558e8ecd18b2efe94923229d02. This does reintroduce a small amount of duplication, but unfortunately that is unavoidable with the current setup.